components/vector-kubearchive-log-collector/production/stone-prod-p02/loki-helm-values.yaml

```diff
@@ -0,0 +1,266 @@
+---
```

Comment:
This file has unneeded config that it's going to be overwritten by `loki-helm-prod-values.yaml`. Can you please make sure that this file has exactly the same contents as`staging/stone-stg-rh01/loki-helm-values.yaml` and `development/loki-helm-values`? The three files should be identical.

Remember that after the refactor I made the idea is that for each generator we should have:
- `<component>-helm-values.yaml`  --> with exactly the same content for every cluster
- `<component>-helm-<env>-valyes.yaml --> should be identical for all the clusters under staging, and identical for all the clusters under production but potentially different between staging and production.